modules/ckanext-advancedsearch/ckanext/advancedsearch/controller.py

```diff
@@ -0,0 +1,45 @@
+import ckan.lib.base as base
+from ckan.common import c
+from ckan.lib import helpers as h
+from ckan.lib.base import render
+from ckan.logic import get_action
+
+
+class YtpAdvancedSearchController(base.BaseController):
```

Comment:
Discussed this with you in the daily today. We decided that it can be released like this, but in the future probably move to use the new views because CKAN is moving towards them.